nio/client/base_client.py

```diff
@@ -940,6 +940,14 @@ def add_event_callback(self, callback, filter):
         self.event_callbacks.append(cb)
 
     def add_ephermeral_callback(self, callback, filter):
+        """
+        Deprecated: typo in function name
+        """
+        logger.warn(
```

Comment:
I think we should use the warnings module for this:
https://docs.python.org/3/library/warnings.html

Reply:
Yes that sounds plausible. I tried to use your logging system but according to best practises warnings would be better. I'll adjust it.